frontend/apps/pwa/src/reducers/recorder.ts

```diff
@@ -1,54 +1,68 @@
-import { createAsyncThunk, createEntityAdapter, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import {
+  createAsyncThunk,
+  createEntityAdapter,
+  createSlice,
+  PayloadAction,
+} from '@reduxjs/toolkit';
 import { v4 as uuidv4 } from 'uuid';
-import { FieldValue, FormDefinition, Record } from 'core-api-client';
+import { FormDefinition, Record } from 'core-api-client';
 
-import { RootState } from '../../app/store';
-import { FormInterface, selectFormOrSubFormById, selectRootForm } from '../../reducers/form';
-import { recordGlobalSelectors } from '../../reducers/records';
-import client from '../../app/client';
+import { RootState } from '../app/store';
+import client from '../app/client';
 
-export interface FormValue {
+import { FormInterface, selectFormOrSubFormById, selectRootForm } from './form';
+import {recordGlobalSelectors} from "./records";
+
+export interface FormValue extends Omit<Record, 'databaseId'> {
```

Comment:
this type felt redundant, opinions welcome ^^

Reply:
I find this a little difficult to interpret as we're both extending and omitting. I think this is worth keeping if there is a strong relationship between Record and FormValue. ie. if we add a field to Record we will typically want it in FormValue.